jina/docker/hubio.py

```diff
@@ -264,6 +272,7 @@ def build(self) -> Dict:
             result = {
                 'name': getattr(self, 'canonical_name', ''),
                 'path': self.args.path,
+                'host_info': _host_info,
```

Comment:
```suggestion
                'host_info': _host_info if args.host_info else ''
```

Reply:
add a new arg in hub build CLI `--host-info (action=store_true)`

Reply:
Done